assets/js/components/BatchEdit/About/Confirmation.jsx

```diff
@@ -0,0 +1,229 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import {
+  CONTROLLED_METADATA,
+  UNCONTROLLED_METADATA,
+  PHYSICAL_METADATA,
+  RIGHTS_METADATA,
+  IDENTIFIER_METADATA,
+  OTHER_METADATA,
+} from "../../../services/metadata";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import UIFormField from "../../UI/Form/Field";
+import UIFormInput from "../../UI/Form/Input";
+import { toastWrapper } from "../../../services/helpers";
+
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
+
+const addWrapperCss = css`
+  border: 1px solid green;
+`;
+
+const removeWrapperCss = css`
+  border: 1px solid red;
+`;
+
+const confirmationNote =
+  'NOTE: This will affect all works currently selected. Please proceed with extreme caution. To execute this change, type "I understand"';
+
+const deleteMetadata = {
+  contributor: [
+    {
+      role: { scheme: "MARC_RELATOR", id: "aut" },
+      term: "http://id.loc.gov/authorities/names/n79091588",
+    },
+  ],
+  genre: [{ term: "http://vocab.getty.edu/aat/300026031" }],
+  description: "123 this is set to be removed",
+};
+
+const BatchEditConfirmation = ({
+  addMetadata,
+  removeMetadata,
+  isModalOpen,
+  handleClose,
+}) => {
+  const [confirmationError, setConfirmationError] = useState({});
+  const [parsedAddMetadata, setParsedAddMetadata] = useState();
+  const [parsedDeleteMetadata, setParsedDeleteMetadata] = useState();
+
+  const metadataItems = CONTROLLED_METADATA.concat(
+    UNCONTROLLED_METADATA,
+    PHYSICAL_METADATA,
+    RIGHTS_METADATA,
+    IDENTIFIER_METADATA,
+    OTHER_METADATA
+  );
+
+  useEffect(() => {
+    let parsedAddData = {};
+    Object.keys(addMetadata).map((key) => {
+      let obj = metadataItems.find((item) => item.name === key);
+      parsedAddData[key] = { ...obj, metadata: addMetadata[key] };
+    });
+    setParsedAddMetadata(parsedAddData);
+
+    let parsedDeleteData = {};
+    Object.keys(deleteMetadata).map((key) => {
+      let obj = metadataItems.find((item) => item.name === key);
+      parsedDeleteData[key] = { ...obj, metadata: deleteMetadata[key] };
+    });
+    setParsedDeleteMetadata(parsedDeleteData);
+  }, []);
+
+  const handleConfirmationChange = (e) => {
+    const filterValue = e.target.value;
+    filterValue == "I understand"
+      ? setConfirmationError()
+      : setConfirmationError({
+          confirmationText: "Confirmation text is required.",
+        });
+  };
+
+  const handleBatchEditConfirm = () => {
+    toastWrapper(
+      "is-success",
+      "Form successfully submitted.  Check the console for form values."
+    );
+    handleClose();
+  };
+
+  return (
+    <div
+      className={`modal ${isModalOpen ? "is-active" : ""}`}
+      data-testid="modal-confirmation"
```

Comment:
We're going to have a few modals here...mind updating this to "modal-batch-edit-confirmation" ?